Intersect.Server.Core/Entities/Player.cs

```diff
@@ -1791,12 +1802,17 @@ public void AdminWarp(Guid newMapId, float newX, float newY, Guid newMapInstance
                 if (!newMap.TryGetInstance(MapInstanceId, out newMapInstance))
                 {
                     // Create a new instance for the map we're on
-                    newMap.TryCreateInstance(MapInstanceId, out newMapInstance);
+                    newMap.TryCreateInstance(MapInstanceId, out newMapInstance, this);
+                    // ...and its surroundings
                     foreach (var surrMap in newSurroundingMaps)
                     {
-                        MapController.Get(surrMap).TryCreateInstance(MapInstanceId, out var surrMapInstance);
+                        MapController.Get(surrMap).TryCreateInstance(MapInstanceId, out _, this);
```

Comment:
Use `MapController.TryGet` to prevent an NRE here

Reply:
done